server/src/models/run.js

```diff
@@ -727,6 +729,18 @@ module.exports = {
 		if (queryParams.order) {
 			if (queryParams.order === 'date')
 				queryOptions.order = [['createdAt', 'DESC']];
+			if (queryParams.order === 'time')
+				queryOptions.order = [[sequelize.literal('ticks * tickRate'), 'ASC']];
+		}
+		if (queryParams.mapName) {
+			queryOptions.include.push({
+				model: Map,
+				as: 'map',
+				where: { name: {[Op.startsWith]: queryParams.mapName} }
+			});
+		}
+		if (queryParams.isPB) {
+			queryOptions.include[1].required = queryParams.isPB; // Works, but feels wrong to assume the array index directly like this
```

Comment:
I believe you could instead put this directly in the include object, with it like:
```
					attributes: ['rank'],
					required: queryParams.isPB ?? false,
```

Reply:
JS didn't like `??` and would throw an `Uncaught SyntaxError: Unexpected token '?'` but using `||` should work in this case since if `isPB` is false and not null it would still return the expected false result. Changed in b1bd918